agave_pyclient/agave_pyclient/agave.py

```diff
@@ -155,6 +155,13 @@ class AgaveRenderer:
     AgaveRenderer communicates with AGAVE running in server mode to perform GPU volume
     rendering.
 
+    Parameters
+    ----------
+    url: str
+        Full url to websocket server including port
+    mode: str
+        "pathtrace" or "raymarch" (pathtrace is default)
```

Comment:
am I not seeing the code that gracefully fails for modes like 'path trace'?

Reply:
if you pass in an unrecognized mode name it will use "pathtrace".  That's handled on the other end where it decodes the string, renderlib/renderlib.cpp line 556.  At the python layer the string is just propagated through the pipe as-is.  I'll add some Python Typing stuff to decorate the function argument here.